src/oscar/templates/oscar/customer/wishlists/wishlists_form.html

```diff
@@ -38,6 +40,16 @@
                 {% endblocktrans %}
             </p>
         {% endif %}
+
+        <div class="wishlist-shared-emails mt-5">
+            <h3>{% trans "Share wishlist" %}</h3>
+            <small>
+                {% trans "If you want to share your wishlist with other accounts, you can add their emails below. The visibility must be set to shared in order for this to work." %}
```

Comment:
Please amend the first sentence to: "If you want to share your wishlist with others, you can add their email addresses below."